conda-recipe/build.sh

```diff
@@ -83,10 +94,11 @@ if [ ! -d "${INSTALLED_DIR}/imex/lib" ]; then
         -DLLVM_ENABLE_ZSTD=OFF \
         -DLLVM_ENABLE_ZLIB=OFF \
         -DLLVM_EXTERNAL_PROJECTS="Imex"  \
-        -DLLVM_EXTERNAL_IMEX_SOURCE_DIR=. \
-        -DLEVEL_ZERO_DIR=${INSTALLED_DIR}/level-zero \
-        -DIMEX_ENABLE_SYCL_RUNTIME=1 \
-        -DIMEX_ENABLE_L0_RUNTIME=1
+        -DLLVM_EXTERNAL_IMEX_SOURCE_DIR=.
+        # -DLEVEL_ZERO_DIR=${INSTALLED_DIR}/level-zero \
+        # -DIMEX_ENABLE_SYCL_RUNTIME=1 \
+        # -DIMEX_ENABLE_L0_RUNTIME=1 \
+        # -DCMAKE_CXX_FLAGS="-I${SPIRV_INC_DIR}"  # grrrr
```

Comment:
rm comments?

Reply:
If you don't mind I'll keep the comments for later when we enable oneAPI/devices.